package.json

```diff
@@ -142,9 +142,6 @@
     "build-ios-e2e": "detox build -c ios.sim.release",
     "test-ios-e2e": "detox test -c ios.sim.release --cleanup"
   },
-  "bin": {
```

Comment:
Why don't we want this anymore? It seems like it is there to catch people from making mistakes. Is that mistake not possible anymore? Or we just think it isn't a big deal anymore?

Reply:
We have kept this behaviour, but made `react-native-local-cli` to have "bin" property instead. So just declaring in a different `npm` module (since React Native itself doesn't have an executable anymore)